persist/persist.go

```diff
@@ -0,0 +1,81 @@
+// Package persist implements delegation persistence. It is primarily
+// concerned with configuration and serialisation; encryption and
+// decryption is done by the cryptor package.
+package persist
+
+import (
+	"errors"
+
+	"github.com/cloudflare/redoctober/config"
+	"github.com/cloudflare/redoctober/keycache"
+	"github.com/cloudflare/redoctober/passvault"
+)
+
+var defaultStore Store = &File{}
+
+const (
+	// NeverPersist indicates that the persistence store will
```

Comment:
Looks like you changed the name of this from `NeverPersist` to `Disabled`, but not here?